src/vivarium_csu_ltbi/tools/builder.py

```diff
@@ -138,6 +118,10 @@ def write_demographic_data(artifact, location):
     key = 'cause.all_causes.cause_specific_mortality_rate'
     write(artifact, key, load(key))
 
+    # TODO - placeholder
+    key = f'cause.{TUBERCULOSIS_AND_HIV}.cause_specific_mortality_rate'
+    write(artifact, key, data.csmr_300)
```

Comment:
Yongquan says this is "csmr_298 + csmr_297 (CSMR for the whole model!)"

Reply:
It's in the updated word doc with the tables of calculations

Reply:
Good, change made